kerdock.yml

```diff
@@ -0,0 +1,51 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# code id, physical, logical are all lower case
+code_id: kerdock
+physical: bits
+logical: bits
+
+name: 'kerdock'
+introduced: '\url{https://doi.org/10.1016/S0019-9958(72)90376-2}'
+
+description: 'Kerdock code is a \([[2^m, 2^{2m}, 2^{m-1} - 2^{frac{m}{2}-1}]]\) binary code where \(m\) is even. The size of code book is \(|2^{m-1}||R(1,m)|\) where \(R(1,m)\)
+is the corresponding first order Reed Muller code. The code is non linear and can exist for perimeters for which there is no linear code (See Nordstrom-Robinson). The relative
```

Comment:
Replace "The code is non linear and can exist for perimeters for which there is no linear code (See Nordstrom-Robinson)." 
with "The code is nonlinear, and its size is twice the size of the largest possible linear code with the same length and distance".